torchinductor/compile_fx.py

```diff
@@ -40,36 +36,17 @@ def disable(obj):
         return False
 
 
-class CheckEachNode(torch.fx.Interpreter):
-    def call_function(self, target, args, kwargs):
-        expected = target(*args, **kwargs)
-        if target in (operator.getitem,):
-            return expected
-
-        gm, gm_inps = gen_gm_and_inputs(target, args, kwargs)
-        graph = GraphLowering(gm)
-        with V.set_graph_handler(graph):
-            graph.run(*args, **kwargs)
-            actual = graph.compile_to_fn()(*gm_inps)
-
-        if isinstance(expected, torch.Tensor):
-            actual = actual[0]
-
-        print(target, same(expected, actual))
-        assert same(expected, actual)
-
-        return expected
-
-
-@functools.partial(wrap_debug, compiler_name="inductor")
+@DebugContext.wrap
 def compile_fx_inner(
     gm: torch.fx.GraphModule,
     example_inputs: List[torch.Tensor],
     wrap=identity,
     cudagraphs=None,
     num_fixed=0,
+    is_backward=False,
 ):
-    init_logging()
+    log.info("Compiling %s graph", "BACKWARDS" if is_backward else "FORWARDS")
+    V.debug.fx_graph(gm, example_inputs)
```

Comment:
Check the config.fx_graph flag here?

Reply:
Oh we do this through `__getattr__` - maybe leave a comment on the class to that end? it wasn't readily obvious.